lars-reimann/api-editor
# feat: @boundary

Closes #175 

### Summary of Changes
Adds boundary annotation on parameter level

### Testing instructions
Create / delete / export / import boundary annotations. 
Check form validation during execution (now also doesn't allow unrestricted intervals)
